src/views/activity/activity_item_view.js

```diff
@@ -20,5 +26,26 @@ export default BaseView.extend({
 
   sendSelectionDetails (event) {
     webChannel.sendUrlSelected(this.model.url, 'url');
+  },
+
+  injectFavicon () {
+    // Manually construct the favicon element and inject it favicon into the
+    // DOM, allowing us to delegate events to its onLoad event, which is not
+    // possible through Ampersand since load events do not bubble.
```

Comment:
This comment seems to be out of date. By directly setting listeners DOM Level 0 style, i.e., by assigning the function to the object, we don't need to delegate events.